MatrixSDK/Contrib/Swift/MXSession.swift

```diff
@@ -275,4 +275,28 @@ public extension MXSession {
         return __unIgnoreUsers(userIds, success: currySuccess(completion), failure: curryFailure(completion))
     }
     
+    
+    /**
+     Register a global listener to events related to the current session.
+     
+     The listener will receive all events including all events of all rooms.
+     
+     - parameters:
+        - types: an array of event types to listen to
+        - block: the block that will be called once a new event has been handled.
+        - returns: a reference to use to unregister the listener
```

Comment:
Good catch! This is fixed in https://github.com/matrix-org/matrix-ios-sdk/pull/327/commits/17383a2b0138f71e5b4bd2c3e235afa9ab978dfb